jquery.signaturepad.js

```diff
@@ -429,11 +427,17 @@ function SignaturePad (selector, options) {
       })
 
       canvas.each(function () {
-        this.ontouchstart = null
+        this.removeEventListener('touchstart', touchStartProcedure)
       })
     }
   }
+  function touchStartProcedure(e) {
```

Comment:
@hassaanalansary how about something like `startTouchDrawing`? We've got `startDrawing` and `stopDrawing` methods, so at least it's consistent with those.

Reply:
Changed the name